packages/google_sign_in/extension_google_sign_in_as_googleapis_auth/lib/extension_google_sign_in_as_googleapis_auth.dart

```diff
@@ -15,15 +15,20 @@ import 'package:http/http.dart' as http;
 /// client that can be used with the rest of the `googleapis` libraries.
 extension GoogleApisGoogleSignInAuth on GoogleSignIn {
   /// Retrieve a `googleapis` authenticated client.
-  Future<googleapis_auth.AuthClient> authenticatedClient({
-    @visibleForTesting GoogleSignInAuthentication debugAuthentication,
-    @visibleForTesting List<String> debugScopes = const [],
```

Comment:
This was indeed a bug! Given the amount of use this package is getting (close to none), I'm tempted of moving this out of flutter/plugins and into flutter/packages, so it doesn't interfere more with important parts of the repo moving forward :/

Reply:
I was wondering why it was here. Moving anything that's not actually a plugin to flutter/packages SGTM. (See also https://github.com/flutter/flutter/issues/76867)

Reply:
Yeah, good one too. These should be moved to flutter/packages. I'll try to get it done while preserving history after I'm done with the nnbd _web stuff.